pkg/exasol-rest-api/application_properties.go

```diff
@@ -0,0 +1,81 @@
+package exasol_rest_api
+
+import (
+	"errors"
+	"os"
+)
+
+type ApplicationProperties struct {
```

Comment:
Let's move this idea to the ticket, it's a lot of refactoring

Reply:
https://github.com/exasol/exasol-rest-api/issues/12

Reply:
MAybe you can create a milestone for the first release and add the code quality issues there?
For me, it's ok if you move review findings to issues as long as we can make sure that they are addressed before the release.
I would like to avoid that we create issues and then just skip them and some when delete them.
That's ok for features but not for issues concerning code quality like for example error codes.

Reply:
Milestone is a good idea, I was also thinking about it.